rmarkdown/The Katak Detector.Rmd

```diff
@@ -0,0 +1,105 @@
+ii.   This allows scientists not only to learn which populations of frog species are most at risk, but it also helps them protect frog populations and, crucially, their entire ecosystems.
+
+- Target users
+i.    Researchers & scientist
+ii.   Governmental or non-profit organization
+iii.    Law makers
+
+
+## Data Science Methodology
+
+- Formulating questions
+  i.	Choose domain 
+  ii.	Identify the current problem we are currently facing
+- Data gathering
+  i.	Consider many data resources such as github, Kaggle, EY
+- Data Cleaning
+  i.	Drop irrelevant column
+  ii.	Identify the missing values and decide on what to do with it
+- EDA & Data visualization
+  i.	Plot the trends using line graph and geospatial mapping
+  ii.	Attempt to answer the initial questions
+- Modelling & Prediction
+- Deployment
+
+Result:
+
+To be able to develop a monitoring tool of frog occurrence for numbers of identified frog species across Australia, extract useful insight from it and project the prediction of frog occurrence. 
+
+Shinyapps:
+
+>https://yuenherny.shinyapps.io/thekatakdetector/
+
+
+## Conclusions
+
+a.    Ultimately, frogs are important to the world because they are essentially our “canary in the coal mine” when it comes to the environment. They are considered natural bioindicators, which is something that measures the health of the environment. Where frogs occur, we see healthy, thriving, resilient ecosystems.
+b.    Monitoring indicator species can also better help scientists their research on ecology and biodiversity. For example, while a wetland ecosystem may show general signs that something is amiss, if scientists see that frog populations are plummeting, they’ll know the problem may be pesticides.
+c.    Scientists, lawmakers, and public officials can also use the data collected from indicator species to implement conservation policies. For examples, protecting land and habitats, regulating development or chemical use, or passing laws to reverse or prevent damage before it becomes too late.
+
+- Project challenges & limitations
+
+Shinyapp allocates only maximum size of 1 GB for the Free and Starter plans
+
+i.    Reduce the data up to 2020 since it can't handle more than 44k rows of data (Original data has 193792k rows)
+ii.   Shinyapps keeps on crashing & disconnected from server due to too many data & memory usage of libraries used
+iii.    Due to computation limits, we omitted the usage of mapview library as it consumed too much memory
+iv.   Unable to showcase prediction outcome, since we dropped huge chunk of data
+
+- Future recommendations
+
+Build a computational model that can predict the count of frogs for a specific location using multiple datasets, and to validate model on other locations of other countries using a variety of open-source geospatial datasets available on Microsoft Planetary Portal (TerraClimate, Sentinel L-2, Copernicus Elevation, Esri 10m Land Cover and JRC Water Surface).

```

Comment:
Maybe we can add another recommendation:
- Host the data in a separate database to reduce Shiny server load.